### ISSUE TITLE
Feedstock conversion failures

### ISSUE THREAD

**Issue by jakirkham** [conda-forge/staged-recipes#179]
Possibly a collision here. Restarting one of them.

* https://travis-ci.org/conda-forge/staged-recipes/jobs/117766307#L258
* https://travis-ci.org/conda-forge/staged-recipes/jobs/117767354#L308

**Comment by jakirkham** [comment]
Retrying made no difference. Still had the same error.

**Comment by jakirkham** [comment]
Thoughts, @pelson? Seems to be unable to add the git remote. Does it already exist somehow?

**Comment by ocefpaf** [comment]
The error is:

```shell
Calculating the recipes which need to be turned into feedstocks.
Making feedstock for rank_filter
/Users/travis/build/conda-forge/staged-recipes/recipes/rank_filter is in fine form
'numpy x.x' requires external setting
fatal: Not a git repository (or any of the parent directories): .git
Traceback (most recent call last):
  File ".CI/create_feedstocks.py", line 106, in <module>
    cwd=feedstock_dir)
  File "/Users/travis/miniconda/lib/python2.7/subprocess.py", line 540, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['git', 'remote', 'add', 'upstream_with_token', 'https://conda-forge-admin:a73335bc94e71106960992fbddfc7f910a2c5a1a@github.com/conda-forge/rank_filter-feedstock']' returned non-zero exit status 128
```

Check if there are changes in `conda-build-all` that led to this.

**Comment by jakirkham** [comment]
The last change was adding R and Perl support. ( https://github.com/SciTools/conda-build-all/pull/27 )

**Comment by ocefpaf** [comment]
Last time we had green we used `conda-smithy-0.5.1` instead of the latest `conda-smithy-0.6.0`. Checking now what is wrong. We can enforce a downgrade meanwhile.

**Comment by jakirkham** [comment]
So, I have tried clearing out the repos, which were already converted to feedstocks. Let's see if it can convert the remaining too.

**Comment by jakirkham** [comment]
I'll add a PR.

**Comment by jakirkham** [comment]
PR to pin conda-smithy is here ( https://github.com/conda-forge/staged-recipes/pull/181 ).

**Comment by jakirkham** [comment]
Yep, still need it as this is failing still ( https://travis-ci.org/conda-forge/staged-recipes/jobs/117783695#L258 ).

**Comment by jakirkham** [comment]
Could be related to a `conda-smithy` syntax change. ( https://github.com/conda-forge/conda-smithy/compare/v0.5.1...v0.6.0#diff-04c6e90faac2675aa89e2176d2eec7d8R38 )

**Comment by jakirkham** [comment]
We have determined that some bug entered between 0.5.1 and 0.6.0 of `conda-smithy`. Will be looking at `git bisect` to identify the change that caused it.

**Comment by jakirkham** [comment]
I assume that is you, @pelson, hiding behind the machine. :wink:

**Comment by pelson** [comment]
Aha, yes, github logins confuse me :smile:

**Comment by jakirkham** [comment]
No worries. :smile:

**Comment by pelson** [comment]
Problem was related to conda-build's need to have CONDA_NPY set to even parse the recipe yaml. I worked-around the problem in conda-smithy and released a v0.6.1. Things are looking rosy again (and we have appveyor scripts which are empty where appropriate).

**Comment by jakirkham** [comment]
Thanks. I saw this fix. LGTM.
